docs/networking/simpleimpl.md

```diff
@@ -98,21 +115,19 @@ Sending Packets
 
 ### Sending to the Server
 
-There is but one way to send a packet to the server. This is because there is only ever *one* server the client can be connected to at once. To do so, we must again use that `SimpleChannel` that was defined earlier. Simply call `INSTANCE.sendToServer(new MyMessage())`. The message will be sent to the handler for its type, if one exists.
+There is but one way to send a packet to the server. This is because there is only ever *one* server the client can be connected to at once. To do so, we must again use that `SimpleChannel` that was defined earlier. Simply call `INSTANCE.send(new MyMessage(),PacketDistributor.SERVER.noArg())`. The message will be sent to the handler for its type, if one exists.
 
 ### Sending to Clients
 
-Packets can be sent directly to a client using the `SimpleChannel`: `HANDLER.sendTo(new MyClientMessage(), serverPlayer.connection.getConnection(), NetworkDirection.PLAY_TO_CLIENT)`. However, this can be quite inconvenient. Forge has some convenience functions that can be used:
-
```

Comment:
Probably should preface with these are some examples of sending packets to the client.